packages/@misk/simpleredux/.gitignore

```diff
@@ -13,3 +13,4 @@ dist
 lib
 .hash
 .old_build_files
+package-lock.json
```

Comment:
I find this a strange thing to have been added. Is it something you manually added?

Reply:
It kept showing up in untracked files, and since it hadn't ever been committed, I figured there was a reason for that. I can undo this change if you want though.